app/components/organisms/AuthTools.tsx

```diff
@@ -1,14 +1,35 @@
+import Avatar from "@material-ui/core/Avatar";
+import deepOrange from "@material-ui/core/colors/deepOrange";
+import { withStyles } from "@material-ui/core/styles";
 import Link from "next/link";
 import React from "react";
 import { UserConsumer } from "../contexts/user";
 
-const AuthMenu: React.SFC = () => (
+const styles = {
+  orangeAvatar: {
+    backgroundColor: deepOrange[500],
+    color: "#fff",
+    margin: 10,
+  },
+  row: {
+    display: "flex",
+    justifyContent: "center",
+    marginLeft: "auto",
+  },
+};
+
+const AuthMenu: React.SFC<{ classes: any }> = ({ classes }) => (
   <UserConsumer>
     {({ session }) => {
       console.info("session", session);
       return (
-        <div>
-          {(session && session.user && session.user.email) || (
+        <div className={classes.row}>
+          {(session &&
+            session.user && (
+              <Avatar className={classes.orangeAvatar}>
+                {session.user.email.slice(0, 2)}
```

Comment:
Made an assumption here that if there's a user they will have an email address greater than 2 character haha

Reply:
Pretty sure that's a safe assumption. Plus anyway, slicing anything smaller than the slice gets you the entire thing, so like `'w'.slice(0, 2) === 'w'` and `''.slice(0, 2) === ''`.